### ISSUE TITLE
[rllib]how to set the tf.keras BN layer's training=True or False in rllib model

### ISSUE THREAD

**Issue by falsemail** [ray-project/ray#11199]
### Describe your feature request
When using rllib（for example PPO） for long time training, it is often stopped because of Nan or Inf in the output actions, and I suspect that long training time results in too large weights of the policy model, so I want to add the BN(Batch Normalization) layer to the policy model. 
My question is, how does the BN layer in t f.keras are embodied in the rllib policy model, and is there a living example of the relevant documentation, about how to set the training=True or False in rllib model? Thank you so much.
For feature requests or questions, post on our Github Discussions page instead: https://github.com/ray-project/ray/discussions!
